doc/website/getting-started/list-of-examples.md

```diff
@@ -0,0 +1,16 @@
+# List of examples
+
+| Example                                         | Description                                                 | Level              |
+|:------------------------------------------------|:------------------------------------------------------------|:-------------------|
+|[icehello](./examples/icehello/)                 | How to send data to another process                         | :star:             |
+|[icedelivery](./examples/icedelivery/)           | How to send and receive data in a functional way            | :star:             |
+|[iceoptions](./examples/iceoptions/)             | Configure the history cache size and the starting behaviour | :star:             |
+|[icedelivery_in_c](./examples/icedelivery_in_c/) | How to send and receive data using the C API                | :star:             |
+|[callbacks](./examples/callbacks/)               | Implement callbacks which are triggered by events           | :star::star:       |
+|[callbacks_in_c](./examples/callbacks_in_c/)     | How to implement callbacks in C                             | :star::star:       |
+|[waitset](./examples/waitset/)                   | Wait until one or more subscriber has received data         | :star::star:       |
+|[waitset_in_c](./examples/waitset_in_c/)         | How to use the WaitSet in C                                 | :star::star:       |
+|[iceensemble](./examples/iceensemble/)           | How to make use of the n:m communication feature            | :star::star:       |
+|[singleprocess](./examples/singleprocess/)       | How to communicate in a single process between threads      | :star::star:       |
+|[iceperf](./examples/iceperf/)                   | Measures the latency of different IPC mechanisms            | :star::star::star: |
+|[icecrystal](./examples/icecrystal/)             | How to use the introspection client                         | :star::star::star: |
```

Comment:
How to use the introspection client for debugging the middleware

Reply:
I'll leave out "the middleware" as this does not add value when reading over the list. In the context of iceoryx this should be clear.

Reply:
Using the introspection to monitor the system state (or similar)